src/constants/earlyRequestMagicNumber.ts

```diff
@@ -1,9 +1,24 @@
+import { bigNumberFromFloatString } from "@shared/utils";
+
 export const earlyRequestMagicNumber =
   "-57896044618658097711785492504343953926634992332820282019728.792003956564819968";
 export const earlyRequestMagicNumberWei =
   "-57896044618658097711785492504343953926634992332820282019728792003956564819968";
 
-export function isEarlyVote(value: string | undefined): boolean {
-  if (value === undefined) return false;
-  return [earlyRequestMagicNumber, earlyRequestMagicNumberWei].includes(value);
+/**
+ * Description
+ * @param {string} value this string is most likely formatted as localeString (includes commas, to a precision of 2)
+ * This function compares only the integer part of earlyRequestMagicNumber and param "value"
+ */
+export function isEarlyVote(value: string | undefined | null): boolean {
+  if (!value) return false;
+
+  const asBigInt = takeIntegerAsBigNumber(value)
+ return [earlyRequestMagicNumber, earlyRequestMagicNumberWei].map(takeIntegerAsBigNumber).some(num => num.eq(asBigInt))
 }
+
+function takeIntegerAsBigNumber(value: string) {
+  // remove commas, if present, remove decimals, take only integer
+  return bigNumberFromFloatString(value.replace(/,/g, "").split(".")[0])
+}
+
```

Comment:
only compare the integer part of the numbers, because the value we get from the proposal data is formatted (has comma separators and is only to a precision of 2)